src/main.rs

```diff
@@ -13,8 +13,8 @@ use ic_nervous_system_common::{serve_logs, serve_logs_v2, serve_metrics};
 
 use evm_rpc::*;
 
-#[ic_cdk_macros::update]
-#[candid_method]
+#[update(name = "eth_getLogs")]
+#[candid_method(rename = "eth_getLogs")]
```

Comment:
Question for anyone more familiar with the Rust CDK: is there a way to avoid all these duplicate attributes (aside from renaming the function itself)?

Reply:
I don't know the answer. What happens if you only use `rename`?

Reply:
Only using one or the other results in the generated Candid interface having a different name from the canister method.